Snowplow iOSTests/Global Contexts/TestGlobalContexts.m

```diff
@@ -57,29 +57,18 @@ @interface TestGlobalContexts : XCTestCase
 @implementation TestGlobalContexts
 
 - (void)testGlobalContexts {
-    SPEmitter *emitter = [SPEmitter build:^(id<SPEmitterBuilder> builder) {
-        [builder setUrlEndpoint:@"com.acme.fake"];
-    }];
-    SPSubject *subject = [[SPSubject alloc] initWithPlatformContext:YES andGeoContext:NO];
-    
     SPGlobalContext *staticGC = [[SPGlobalContext alloc] initWithStaticContexts:@[[[SPSelfDescribingJson alloc] initWithSchema:@"schema" andData:@{@"key": @"value"}]]];
     SPGlobalContext *generatorGC = [[SPGlobalContext alloc] initWithContextGenerator:[GlobalContextGenerator new]];
     SPGlobalContext *blockGC = [[SPGlobalContext alloc] initWithGenerator:^NSArray<SPSelfDescribingJson *> *(id<SPInspectableEvent> event) {
         return @[
             [[SPSelfDescribingJson alloc] initWithSchema:@"schemaBlock" andData:@{@"key": @"value"}],
         ];
     }];
-    
-    SPTracker *tracker = [SPTracker build:^(id<SPTrackerBuilder> builder) {
-        [builder setEmitter:emitter];
-        [builder setSubject:subject];
-        [builder setAppId:[NSUUID UUID].UUIDString];
-        [builder setGlobalContextGenerators:@{
-            @"static": staticGC,
-            @"generator": generatorGC,
-            @"block": blockGC,
-        }];
-    }];
+    SPTracker *tracker = [self getTrackerWithGenerators:@{
```

Comment:
getTrackerWithGenerators isn't particularly clear imo. Maybe `getTrackerWithGlobalContextGenerators` is better, bit long but worth it.